### ISSUE TITLE
Allow to quickly switch between accounts

### ISSUE THREAD

**Issue by fallsdevil** [thunderbird/thunderbird-android#7260]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.603

### Problem you are trying to solve

It's easier to switch email accounts, in the Gmail app you can change the account just by sliding the account photo up or down, this is useful for those who have several accounts logged in.

### Suggested solution

A slight interface redesign to add the account icon in the top corner to add this swipe feature to change email account.

### Screenshots / Drawings / Technical details

I made this demonstration video to better understand the suggested function.

[Video here](https://imgur.com/a/l8PRsU9).

**Comment by cketti** [comment]
Making it easier to switch between accounts is something we want to enable. We're not sure yet whether that means implementing the proposed solution or going a different route. This needs some UX design exploration before we can commit on an approach.

**Comment by MatzeG1t** [comment]
Google has also a different puspose with the account. Its their central ID for a user and they want to give you access to it in all their products as obvious as possible.
From controllability and usability this proposed solution is actually bad. It needs onboarding for the user because its bascially an invisible feature. INvisible features should be used for frequent tasks without available screen space. Like deleting a mail via swipe in your inbox - you dont want a bin icon next to every email.
If you have a unified inbox its even less important to switch accounts.

From my perspective the current way of switching is controlable and easy to learn. An  improvement I'd see would be rather minimal:

I would suggest to switch the account in the menu and maybe animate it a bit nicer (move up, or some sort of exchange animation) and/or hightlight (background) the active account a bit more. Currently only the color changes. But this color wasnt given by me, so it doesnt have any meaning - the accountname (email adress) is sufficient.

**Comment by fallsdevil** [comment]
@MatzeG1t Yes, I agree with you ... This resource of switching sliding to me is useful in case I have several accounts, type 4, or 5, and when I need to receive a verification code I can not find it because when updating the Synchony Emails are above code, so I have difficulty finding it, and in these cases it would be great to be able to switch quickly especially when there is an image in the account icon, because I identify faster since most codes have a time of time Short validity. Even today the current interface shows the initials and colors of the accounts, in certain cases, would still need to enter several accounts until I find the email I need. So I think there can be considerable solution even if it is not necessarily.

**Comment by knomepasi** [comment]
@MatzeG1t Regarding hidden features, there is already one related to switching accounts: when you open the menu and click the account name on the top, the account is switched to the next one.

To me, this is a less unintuitive and harder to reach version of the suggested swipe action and I would happily lose this feature over the updated one.

**Comment by kewisch** [comment]
For the moment our focus on this is #8812. We're considering some improvements to navigation between accounts, therefore I'm duplicating this against that issue. Thank you for reporting and the video!
